Assets/scss/components/article-cards.scss

```diff
@@ -0,0 +1,95 @@
+.article-cards{
+    background-color: $white-smoke;
+    font-family: $arial;
+    max-width: 100%;
+    margin: 0;
+
+    &__menu{
+        display: flex;
+        padding: 20px;
+       
+        &--item{
+            @include article-dropdown-style;
+            max-width: 150px;
+        }
+        .dropdown-up{
+            @include article-dropdown-style;
+        }
+    }
+
+    &__container{
+        max-width: inherit;
+        display: grid;
+        gap: 20px;
+        @include min-screen(sm) {
+            padding: 0px 20px;
+        }
+        @include min-screen(md) {
+            grid-template-columns: repeat(2, 1fr);
+
+        }
+        @include min-screen(lg) {
+            grid-template-columns: repeat(3, 1fr);
+        }
+    }
+    &__card{
+        background-color: var(--card-bg-color);
+        margin-bottom: 20px;
+        &--image{
+            width: 770px;
+        }
+        &--detailed{
+            display: flex;
+            margin-left: 1rem;
+            margin-top: 1rem;
+            .card-detailed__location{
+                font-weight: $font-bold;
+                color: $wine;
+
+                &::after{
+                    content: "   | ";
+                    padding: 0px 8px;
+                }
+
+            }
+            .card--detailed__publishdate{
+                font-size: 12px;
+                font-weight: $font-bold;
+                margin-top: 4px;
+                margin-left: 8px;
+
+            }
+
+        }
+        &--title{
+            color: var(--card--title-color);
+            margin-left: 1rem;
+            font-size: 2rem;
+            margin: 1rem 1rem;
+        }
+        &--description{
+            margin: 0rem 1rem;
+            margin-bottom: 2rem;
+            padding-right: 1rem;
+
+            .description-detail{
+                font-size: 14px;
+                line-height: 22px;
```

Comment:
Line heights can be set relative to font size, in this case 22/14 = 1.571428 so you could set `line-height: 1.571428`

Reply:
Uodated.

Reply:
We don't need the `toRem()` function, as we intend to use it as a [unitless number](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#number) which will scale using the value of `font-size`:

> The used value is this unitless [number](https://developer.mozilla.org/en-US/docs/Web/CSS/number) multiplied by the element's own font size. The computed value is the same as the specified number. In most cases, **this is the preferred way** to set line-height and avoid unexpected results due to inheritance.

So in this case, the value would be exactly `1.571428` because that multiplied by the font size (`toRem(14) = 0.875rem * 16px = 14px`) will be ~22px (not exact, but effectively exact)

Reply:
Got it  @jwetmore-hoodoo  Thanks!